tests/everflow/test_everflow_testbed.py

```diff
@@ -836,6 +836,119 @@ def background_traffic(run_count=None):
             background_thread.join()
             background_traffic(run_count=1)
 
+    def test_everflow_fwd_recircle_port_queue_check(self, setup_info, setup_mirror_session, # noqa F811
+                                                    dest_port_type, ptfadapter, tbinfo,
+                                       toggle_all_simulator_ports_to_rand_selected_tor,     # noqa F811
+                                       setup_standby_ports_on_rand_unselected_tor_unconditionally):    # noqa F811
+        """
+        Verify basic forwarding scenario with mirror session config having specific queue for the Everflow feature.
+        Make sure the mirrored packets are sent via the specific queue on recircle port
+        """
+        duthost_set = BaseEverflowTest.get_duthost_set(setup_info)
+        everflow_dut = setup_info[dest_port_type]['everflow_dut']
+        remote_dut = setup_info[dest_port_type]['remote_dut']
+        remote_dut.shell(remote_dut.get_vtysh_cmd_for_namespace(
+            "vtysh -c \"configure terminal\" -c \"no ip nht resolve-via-default\"",
+            setup_info[dest_port_type]["remote_namespace"]))
+
+        def update_acl_rule_config(table_name, session_name, config_method, rules=everflow_utils.EVERFLOW_V4_RULES):
+            rules_config = everflow_utils.load_acl_rules_config(table_name,
+                                                                os.path.join(everflow_utils.FILE_DIR, rules))
+            rules_config['rules'] = [
+                rule for rule in rules_config['rules']
+                if 'transport' not in rule.get('qualifiers', {}) or
+                   not any('tcp-flags' in x for x in rule.get('qualifiers', {}).get('transport', {}).keys())
+            ]
+            duthost.host.options["variable_manager"].extra_vars.update(rules_config)
+
+            if config_method == everflow_utils.CONFIG_MODE_CLI:
+                duthost.template(src=os.path.join(everflow_utils.TEMPLATE_DIR,
+                                                  everflow_utils.EVERFLOW_RULE_CREATE_TEMPLATE),
+                                 dest=os.path.join(everflow_utils.DUT_RUN_DIR,
+                                                   everflow_utils.EVERFLOW_RULE_CREATE_FILE))
+
+                command = "acl-loader update full {} --table_name {} --session_name {}" \
+                    .format(os.path.join(everflow_utils.DUT_RUN_DIR, everflow_utils.EVERFLOW_RULE_CREATE_FILE),
+                            table_name,
+                            session_name)
+
+                # NOTE: Until the repo branches, we're only applying the flag
+                # on egress mirroring to preserve backwards compatibility.
+                if self.mirror_type() == "egress":
+                    command += " --mirror_stage {}".format(self.mirror_type())
+
+            elif config_method == everflow_utils.CONFIG_MODE_CONFIGLET:
+                pass
+
+            duthost.command(command)
+            time.sleep(2)
```

Comment:
Do we know if sleeping for 2 seconds is enough for all platforms?
Should we wait until we see the logs for the ACL rules created similar to `acl/test_acl.py`?
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/acl/test_acl.py#L757-L760

Reply:
Taken care. Removed manual sleep and added code to check syslog for rules creation.